packages/components/src/mobile/gradient/index.native.js

```diff
@@ -0,0 +1,93 @@
+/**
```

Comment:
I see that a big part of that file comes from the `linear-gradient.native.js` that is removed. I think it is always good to have git history and rename/move the file instead of moving only the content of that file

Reply:
Was trying to get the history back but without success, however, there was only one commit.